neuralhydrology/nh_wandb_sweep.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+"""
+This script configures a sweep and creates it on WandB. 
+
+Use nh_run.py in sweep mode to add an agent to the sweep. An agent will then execute as long as there are configurations
+left to try in the sweep. 
+"""
+import wandb
+
+# This specifies the hyperparameters to sweep over
+sweep_config = {
```

Comment:
I fear that people will read this as the "neuralhydorlogy endorsed HP tuning configuration" and not think properly about what they want to sweep. Can we turn this into an empty dict and add a comment that points to the tutorial?